packages/docs/operators/_request_details.yaml

```diff
@@ -0,0 +1,33 @@
+_ref:
+  path: templates/operators.yaml.njk
+  transformer: templates/operatorsMethodTransformer.js
+  vars:
+    pageId: _request_details
+    pageTitle: _request_details
+    filePath: operators/_request_details.yaml
+    env: Client Only
+    types: |
+      ```
+      (requestId: string): any
+      ```
+    description: |
+      The `_request_details` operator returns detailed information of a request. If the request has not yet been called, or is still executing, the returned value is `null`.
```

Comment:
Does it return `null` if the request is executing?

Reply:
Thanks for picking that up, no it doesn't. I have amended here